internal/controller/nats/controller.go

```diff
@@ -47,8 +48,8 @@ const (
 
 // Reconciler reconciles a Nats object.
 //
-//go:generate mockery --name=Controller --dir=../../../vendor/sigs.k8s.io/controller-runtime/pkg/controller --outpkg=mocks --case=underscore
-//go:generate mockery --name=Manager --dir=../../../vendor/sigs.k8s.io/controller-runtime/pkg/manager --outpkg=mocks --case=underscore
+//go:generate go run github.com/vektra/mockery/v2 --name=Controller --dir=../../../vendor/sigs.k8s.io/controller-runtime/pkg/controller --outpkg=mocks --case=underscore
+//go:generate go run github.com/vektra/mockery/v2 --name=Manager --dir=../../../vendor/sigs.k8s.io/controller-runtime/pkg/manager --outpkg=mocks --case=underscore
```

Comment:
Is `--dir` necessary? Why?

Reply:
it is necessary as this points to a controller in the VENDOR directory (not sure if it is an antipattern, but it was there before, so i did not change it )